modules/tools/testing/src/test/resources/test/usage/assert-test.js

```diff
@@ -26,3 +27,11 @@ exports.testAssertWithMessage = function () {
     t.assertNull(undefined, 'message');
     t.assertNotNull(1, 'message');
 };
+
+exports.testAssertThrows = function () {
+    const s = t.assertThrows(() => {throw 'string!'}, Java.type('java.lang.String'));
+    t.assertEquals('string!', s);
+
+    const e = t.assertThrows(() => {throw new Error('message')}, Error);
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8707331)
```suggestion
    const e = t.assertThrows(() => {throw new Error('message');}, Error);
```